.github/workflows/reusable-azure-ai-translate.yml

```diff
@@ -93,16 +93,45 @@ jobs:
         id: datetime
         run: echo "timestamp=$(date -u '+%B %d, %Y at %H:%M UTC')" >> "$GITHUB_OUTPUT"
 
-      - name: Create Pull Request
+      - name: Create Pull Request to update translations
         if: steps.check_pr.outputs.found == 'false'
-        uses: peter-evans/create-pull-request@271a8d0340265f705b14b6d32b9829c1cb33d45e  # v7.0.8
-        with:
-          token: ${{ secrets.NEWFOLD_ACCESS_TOKEN }}
-          commit-message: Auto-translated PO and JSON files
-          branch: ${{ steps.vars.outputs.BRANCH }}
-          title: Updating translation files on ${{ steps.datetime.outputs.timestamp }}
-          body: |
-            This pull request contains automatic translations of PO and JSON files.
-
-            The workflow run responsible for creating this pull request can be found [here](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).
-          labels: translations
+        env:
+          GH_TOKEN: ${{ secrets.NEWFOLD_ACCESS_TOKEN }}
+          BRANCH_NAME: ${{ steps.vars.outputs.BRANCH }}
+          TIMESTAMP: ${{ steps.datetime.outputs.timestamp }}
+        run: |
```

Comment:
Can we separate this out into the following steps:
- Configure Git identity.
- Create or reset to branch.
- Stage changes
- Commit changes
- Push changes/publish branch.
- Create pull request.

This will make it easier to follow and more clear which part of the process is responsible for a failure.